magento/pwa-studio
# [PWA-229] [DX] VSCode Jest Plugin Broken

## Description

After we added automated testing for service workers, the popular Jest plugin for vscode stopped working, due to the --experimental-worker flag that must be appended to test execution. This plugin essentially does what test:dev does, runs the whole suite and then watches the file system for changes, but gives you feedback right in the code base.

While it would be okay to give developers instructions on how to fix, it would probably be better to detect if this flag was passed, and not run these tests, instead of just failing.

Error Message
```
Error: Cannot find module 'worker_threads'
Starting Jest in Watch mode failed too many times and has been stopped.
```
## Related Issue





* [[PWA-229](https://jira.corp.magento.com/browse/PWA-229)] [DX] VSCode Jest Plugin Broken
## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Install Jest plugin for VSCode and enable
2. In the Output panel, verify that test runner starts up w/o error and completes

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
